framework/src/governance/lib/datazone/datazone-gsr-msk-datasource.ts

```diff
@@ -0,0 +1,232 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Duration, Stack } from 'aws-cdk-lib';
+import { CfnProjectMembership } from 'aws-cdk-lib/aws-datazone';
+import { Rule, RuleTargetInput, Schedule } from 'aws-cdk-lib/aws-events';
+import { LambdaFunction } from 'aws-cdk-lib/aws-events-targets';
+import {
+  Effect,
+  ManagedPolicy,
+  PolicyDocument,
+  PolicyStatement,
+  Role,
+  ServicePrincipal,
+} from 'aws-cdk-lib/aws-iam';
+import { Function, Runtime, Code } from 'aws-cdk-lib/aws-lambda';
+import { Construct } from 'constructs';
+import { DataZoneGsrMskDataSourceProps } from './datazone-gsr-msk-datasource-props';
+import { TrackedConstruct, TrackedConstructProps } from '../../../utils';
+
+/**
+ * A DataZone custom data source for MSK (Managed Streaming for Kafka) with integration for Glue Schema Registry.
```

Comment:
Can we clarify what we do here, maybe should be along the lines of sync gsr with datazone?